app/scripts/controllers/preferences-controller.ts

```diff
@@ -17,11 +25,90 @@ const testNetworks = {
   [CHAIN_IDS.LINEA_SEPOLIA]: true,
 };
 
+type PreferencesControllerOptions = {
+  networkConfigurations?: Record<string, { chainId: Hex }>;
+  initState?: Partial<PreferencesControllerState>;
+  initLangCode?: string;
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  network?: any;
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  messenger?: any;
+};
```

Comment:
```suggestion

const controllerName = 'PreferencesController';

/**
 * Returns the state of the {@link PreferencesController}.
 */
export type PreferencesControllerGetStateAction = {
  type: 'PreferencesController:getState';
  handler: () => PreferencesControllerState;
};

/**
 * Actions exposed by the {@link PreferencesController}.
 */
export type PreferencesControllerActions = PreferencesControllerGetStateAction;

/**
 * Event emitted when the state of the {@link PreferencesController} changes.
 */
export type PreferencesControllerStateChangeEvent = {
  type: 'PreferencesController:stateChange';
  payload: [PreferencesControllerState, []];
};

/**
 * Events emitted by {@link PreferencesController}.
 */
export type PreferencesControllerEvents = PreferencesControllerStateChangeEvent;

/**
 * Actions that this controller is allowed to call.
 */
export type AllowedActions =
  | AccountsControllerGetAccountByAddressAction
  | AccountsControllerSetAccountNameAction
  | AccountsControllerGetSelectedAccountAction
  | AccountsControllerSetSelectedAccountAction;

/**
 * Events that this controller is allowed to subscribe.
 */
export type AllowedEvents = AccountsControllerChangeEvent;

export type PreferencesControllerMessenger = RestrictedControllerMessenger<
  typeof controllerName,
  PreferencesControllerActions | AllowedActions,
  PreferencesControllerEvents | AllowedEvents,
  AllowedActions['type'],
  AllowedEvents['type']
>;

type PreferencesControllerOptions = {
  networkConfigurations?: Record<string, { chainId: Hex }>;
  initState?: Partial<PreferencesControllerState>;
  initLangCode?: string;
  messenger: PreferencesControllerMessenger;
};
```

I believe the network property can just be removed I don't see it used anywhere.

Reply:
I believe the`network` property can just be removed I don't see it used anywhere.